packages/docs/vite.config.ts

```diff
@@ -8,7 +8,7 @@ const fs = _fs.promises
 export default defineConfig({
   clearScreen: false,
   plugins: [
-    ...(process.env.NETLIFY ? [] : [copyPiniaPlugin()]),
+    // ...(process.env.NETLIFY ? [] : [copyPiniaPlugin()]),
```

Comment:
it would seem the copy plugin shouldn't be triggered anyway locally due to the netlify check, but it was and it was failing.